Tests/Functional/RunnerTest.php

```diff
@@ -2051,7 +2051,8 @@ public function testPermissionsFailedWithoutContainerRootUserFeature()
                         'entry_point' => 'mkdir /data/out/tables/mytable.csv.gz && '
                             . 'chmod 000 /data/out/tables/mytable.csv.gz && '
                             . 'touch /data/out/tables/mytable.csv.gz/part1 && '
-                            . 'echo "value1" > /data/out/tables/mytable.csv.gz/part1'
+                            . 'echo "value1" > /data/out/tables/mytable.csv.gz/part1 && '
+                            . 'chmod 000 /data/out/tables/mytable.csv.gz/part1'
```

Comment:
ale to vyleti na tom touch, takze tyhle radky se uz neprovedou ne?

Reply:
pravda